starter_code/queries.md

```diff
@@ -4,67 +4,153 @@
 
 ### 1. All the companies that it's name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+Filter : {name: "Babelgum"}
+Project : {name:1}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+Filter : {number_of_employees: {$gt:5000}}
+Project : {number_of_employees: 1}
+Limit : 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
 <!-- Your Code Goes Here -->
+Filter :
+Project :
+Sort :
+Collation : 
+Maxtitems : 
+Skip : 
+Limit : 
 
 ### 4. All the companies that had a Valuation Amount of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
 
-<!-- Your Code Goes Here -->
+Filter : {"ipo.valuation_amount": {$gte:1000000000}}
```

Comment:
C'est correct, il manque juste le champ "founded_year", ce qui donnerait:
 { $and: [{"ipo.valuation_amount": {$gt: 100000000}}, {founded_year: {$lte: 2010}} ] }